src/modifier/modifier.ts

```diff
@@ -775,6 +775,9 @@ export class BypassSpeedChanceModifier extends PokemonHeldItemModifier {
 
     if (!bypassSpeed.value && pokemon.randSeedInt(10) < this.getStackCount()) {
       bypassSpeed.value = true;
+      if (this.type instanceof ModifierTypes.PokemonHeldItemModifierType && this.type.id === "QUICK_CLAW") {
+        pokemon.scene.queueMessage(getPokemonMessage(pokemon, " used its quick claw to move faster!"));
```

Comment:
Should probably be localized

Reply:
unless we remove the pokemon prefix coming from the `getPokemonMessage()` and place the string to a regular i18n locale translation, I dont think this is possible to be localized. please see `src/messages.ts`

so we have 2 options. 
- keep the pokemon prefix but wont be localized
- or remove the pokemon prefix so localization would be possible

Reply:
How about you abstract it to make it work?
```suggestion
        const pokemonPrefix = getPokemonPrefix(pokemon);
        pokemon.scene.queueMessage(i18n.t(:itemUse:quickclaw', {pokemon, pokemonPrefix}));
```

and then in your translations something like
```ts
const de: Record<string, any> = {
  "itemUse:quickclaw": "{{pokemonPrefix}} {{pokemon.name}} used its quick claw to move first!"
}
```

_(just an inspiration, not actual code)_

Reply:
the `getPokemonPrefix()` returns an untranslated text too tho. the whole `src/messages.ts` does not have translation functionality. i think it needs a separate PR for it

Reply:
I agree.
but you can still translate the part you are in control of

Reply:
i cant translate only part of it tho, and i wouldnt wanna touch anything from `messages.ts` on this PR because its being used all over the codebase.
also for reference someone is already working on the translation https://github.com/pagefaultgames/pokerogue/pull/1276/files

Reply:
@Tempo-anon it's now up to you